cf/rds.yaml

```diff
@@ -0,0 +1,123 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Set up RDS Aurora cluster [managed by github.com/hms-dbmi-cellenics/iac]
+
+Parameters:
+  Environment:
+    Type: String
+    Default: development
+    AllowedValues:
+      - development
+      - staging
+      - production
+    Description: The environment for which the cluster needs to be created.
+
+# --------------------------------------- Constants ---------------------------------------
+  DBPort:
+    Type: String
+    Default: "5432"
+
+  DBEngineVersion:
+    Type: Number
+    Default: 13.4
+
+  DBSecretName:
+    Description: The name of the secret used to store master user and password.
+    Type: String
+    Default: aurora-staging
```

Comment:
this needs to be parametrized based on environment

Reply:
Parameter's defaults can't be parametrized, should I remove these parameters? Or only have `aurora` as the `DBSecretName`

Reply:
@ivababukova if I'm not mistaken, this is only the name of the parameter containing the secrets, not the value themselves so I don't think we need to parameterize it.

Reply:
Are we going to have the same secret for both staging and production?

Reply:
I see, in this case it is ok if we don't paramtereize it, but I would remove "staging" from it so that it doesn't cause confusion

Reply:
ok